### PR TITLE
Add cluster

### PR DESCRIPTION
**Pull request by khooroko**
A field for general location. Generated automatically from postal code. Only displayed in infoPanel.

Will be used in the future to list other nearby contacts

### CODE REVIEW

**Review by lawwman — APPROVED** [review]
Very thorough testing for cluster util. Good job.

File: src/main/java/seedu/address/model/util/ClusterUtil.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    public static final String CLUSTER_POSTAL_DISTRICT_10 = "Ardmore, Bukit Timah, Holland Road, Tanglin";
+    public static final String CLUSTER_POSTAL_DISTRICT_11 = "Watten Estate, Novena, Thomson";
+    public static final String CLUSTER_POSTAL_DISTRICT_12 = "Balestier, Toa Payoh, Serangoon";
+    public static final String CLUSTER_POSTAL_DISTRICT_13 = "Macpherson, Braddell";
+    public static final String CLUSTER_POSTAL_DISTRICT_14 = "Geylang, Eunos";
+    public static final String CLUSTER_POSTAL_DISTRICT_15 = "Katong, Joo Chiat, Amber Road";
+    public static final String CLUSTER_POSTAL_DISTRICT_16 = "Bedok, Upper East Coast, Eastwood, Kew Drive";
+    public static final String CLUSTER_POSTAL_DISTRICT_17 = "Loyang, Changi";
+    public static final String CLUSTER_POSTAL_DISTRICT_18 = "Tampines, Pasir Ris";
+    public static final String CLUSTER_POSTAL_DISTRICT_19 = "Serangoon Garden, Hougang, Ponggol";
```
Comment (jaivigneshvenugopal) [inline src/main/java/seedu/address/model/util/ClusterUtil.java]: Punggol*

**Review by jaivigneshvenugopal — APPROVED** [review]
Ponggol to Punggol and its good to merge. Great job buddy interesting way to generate cluster indeed.

**Review by jaivigneshvenugopal — APPROVED** [review]
Really smart way to test cluster indeed!

**Pull request merged**